README.md

```diff
@@ -29,7 +30,17 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+the router defines the CRUD within our app. It allows us to create, read, update, and delete. This CRUD creates six verbs that correlates to our controller.
+
+Get => #index
+Get => #new
+Post => #creates
+Get => #show
+Get => #edit
+Patch/Put => #update
+Delete => #destroy
+
+each of these "verbs" must defined in our controller so requests brought in by the router knows what to do for each.
```

Comment:
the router actually simply sends requests over to the correct controller with the proper action. It does not actually define the verbs